docs/guides/backup_security.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: How to back up your account, restore an account from a backup, export a user's Stronghold, rekey Stronghold.
+description: This page describes how to back up your account, restore an account from a backup, export a user's Stronghold, and rekey Stronghold.
```

Comment:
Tbh I like the old version more.

For SEO the first 3 words are useless (every page is a page and describes something)

For link previews for humans the first 3 words are also useless by the same reasons.

Reply:
🤷👌